components/gitpod-db/src/typeorm/entity/db-project.ts

```diff
@@ -30,7 +30,7 @@ export class DBProject {
         transformer: Transformer.MAP_EMPTY_STR_TO_UNDEFINED,
     })
     @Index("ind_teamId")
-    teamId?: string;
+    teamId: string;
```

Comment:
Here I found:
- 4 projects that are not deleted and don't have this set (SELECT * FROM d_b_project WHERE teamId = '' AND markedDeleted = FALSE;)
- 15k markedDeleted

Would be awesome if we could cleanup that state somehow, so we don't stumble over it in some month's time. :lotus_position:

Reply:
will look into this today

Reply:
I have checked those four projects and they were all unused so I marked them as deleted as well.

Reply:
Thank you! :pray: